### PR TITLE
Testimonial page added and Sponsors routing change

### PR DESCRIPTION
**Pull request by vxd447**

### CODE REVIEW

**Comment by vxd447** [comment]
@saranonearth Unable to find the deployment problem. Can you please look into the matter?

File: src/App.jsx [hunk 1]
```diff
@@ -17,8 +15,9 @@ import { Devs } from "./Pages/Devs";
 import { EventDescription } from "./Pages/EventDescription";
 import { Events } from "./Pages/Events";
 import { Queries } from "./Pages/Queries";
-import { Sponsers } from "./Pages/Sponsers";
+import { Sponsors } from "./Pages/Sponsors";
```
Comment (technobyte-nitkkr) [inline src/App.jsx:18]: 'Sponsors' should be 'sponsors'

Comment (technobyte-nitkkr) [reply src/App.jsx:18]: ./Pages/sponsors

Comment (vxd447) [reply src/App.jsx:18]: I changed the file name to Sponsors ;).

**Comment by saranonearth** [comment]
@technobyte-nitkkr  can post the vercel logs for this PR here? 🚀

**Comment by vxd447** [comment]
Leave this branch I will add these changes in seperate PR. :)

**Comment by vxd447** [comment]
Leave this PR... Changes are already added with another PR

**Pull request closed without merging**